products/samsungmobile.md

```diff
@@ -13,6 +13,27 @@ releaseDateColumn: true
 sortReleasesBy: 'release'
 releases:
 
+  - releaseCycle: "Galaxy A13 5G"
+    release: 2021-12-03
+    support: true
+    eol: false
+    
+  - releaseCycle: "Galaxy F42 5G"
+    release: 2021-10-03
+    support: true
+    eol: false
+    
+  - releaseCycle: "Galaxy M52 5G"
+    release: 2021-10-03
+    support: true
+    eol: false
+    
+  - releaseCycle: "Galaxy M32 5G"
```

Comment:
Upstream lists this as `Galaxy M32` with no 5G. Check please?

Reply:
@Evernow

Reply:
Indeed, my bad that was an oversight